packages/paste-website/src/components/tokens-example/index.tsx

```diff
@@ -117,7 +122,7 @@ export const TokenExample: React.FC<TokenExampleProps> = ({token}) => {
     case 'color':
       return <ColorBox backgroundColor={token.value as any} />;
     case 'border-color':
-      return <ColorBox backgroundColor={tokenName as keyof ThemeShape['borderColors']} />;
+      return <BorderBox borderColor={tokenName as keyof ThemeShape['borderColors']} height='size60' />;
```

Comment:
size60 is rather tall.  Have you checked the preview build? https://paste-git-fork-itsmewiththeface-tokens-border-color.twilio-dsys.now.sh/tokens